src/ontology/config/mondo-excluded-values.yml

```diff
@@ -1,4 +1,9 @@
 exclusions:
+  synonym_exclusion_rules:
+    - rule: ".*otherwise specified"
```

Comment:
## Regex-based lexical exclusion rules
I'm fine w/ this implementation, and with just adding rules incrementally as we go, starting with these.